docs/user/communication.rst

```diff
@@ -41,99 +45,196 @@ Exemplary exercise detail page with posts:
 |exercise-posts|
 
 Course-wide Posts & Course Communication Overview
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------
 
-Besides lecture or exercise related questions, Artemis offers a third post type: posts with course-wide topics, such as ``Organization`` or ``Tech Support``.
-These posts can only be created on the course communication overview, which is shown in the screenshot below.
+Besides lecture or exercise related questions, Artemis offers a third post type: posts with course-wide topics, such as
+``Organization`` or ``Tech Support``. These posts can only be created on the course communication overview, which is shown
+in the screenshot below.
 
 The ``Communication`` space of an Artemis course serves as overview for *all* posts in a course.
 Hence, course-wide posts as well as exercise posts and lecture posts are listed.
 Here, users can easily query, sort, and filter existing posts.
 
 |course-posts|
 
+Secluded Communication
+----------------------
+Users of a course can communicate in private via the Messages page. (see image below) The page consists of a collapsible
```

Comment:
The existing parts of the documentation directly address the user (‘you’ [1,2]). The new parts only do it indirectly (‘Users can […]’).

I think the users should be directly addressed in the documentation written for them. In some cases it might alternatively be okay to just remove the ‘user[s]’ from the middle of a sentence to turn it into a general statement about the offered functionality [3].

This is only the first occurrence I marked, the comment probably applies in a few more locations I did not explicitly comment on as well.